hack/jenkins/job-configs/kubernetes-test-history.yaml

```diff
@@ -0,0 +1,15 @@
+- job:
+    name: kubernetes-test-summary
+    description: 'Create a daily test summary and upload to GCS. Test owner: spxtr.'
+    triggers:
+        - timed: 'H 0 * * *'
+    builders:
+        - shell: |
+            curl -fsS https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/test-history/gen_history > gen_history
+            chmod +x gen_history
+            gen_history http://jenkins-master:8080/
+    publishers:
+        - email-ext:
+            recipients: spxtr@google.com
```

Comment:
I'm not sure if it'd be useful. This emails when the run fails or stops failing, which means I need to go on and figure out why.